tests/Dfc.CourseDirectory.WebV2.Tests/FeatureTests/NewApprenticeshipProvider/ApprenticeshipSummaryTests.cs

```diff
@@ -676,5 +677,85 @@ public async Task PostConfirmation_ValidRequestWithRegionsAndVenue_CreatesValidA
                     l.Regions.Contains("E06000001") &&
                     l.Regions.Contains("E10000009")))));
         }
+
+        [Fact]
+        public async Task PostConfirmation_ValidRequestWithRegionsAndVenue_UpdatesValidApprenticeship2()
+        {
+            // Arrange
+            var ukprn = 12347;
+            var providerUserId = $"{ukprn}-user";
+            var adminUserId = $"admin-user";
+            var contactTelephone = "1111 111 1111";
+            var contactWebsite = "https://somerandomprovider.com";
+            var marketingInfo = "Providing Online training";
+            var regions = new List<string> { "123" };
+            var standardCode = 123;
+            var standardVersion = 1;
+            var providerId = await TestData.CreateProvider(apprenticeshipQAStatus: ApprenticeshipQAStatus.NotStarted);
+            var user = await TestData.CreateUser(providerUserId, "somebody@provider1.com", "Provider 1", "Person", providerId);
+            var adminUser = await TestData.CreateUser(adminUserId, "admin@provider.com", "admin", "admin", null);
+            var standard = await TestData.CreateStandard(standardCode, standardVersion, standardName: "My standard");
+            var apprenticeshipId = await TestData.CreateApprenticeship(providerId,
+                standard,
+                createdBy: user,
+                contactEmail: adminUser.Email,
```

Comment:
Small thing but contact email isn't a property of the user

Reply:
done.